site/src/components/accordion.md

```diff
@@ -0,0 +1,137 @@
+---
+title: Accordion
+description: The accordion component in the Atlas Design System
+template: standard
+# need design to create
+# figmaEmbed: https://www.figma.com/embed?embed_host=share&url=https%3A%2F%2Fwww.figma.com%2Ffile%2FuVA2amRR71yJZ0GS6RI6zL%2F%25F0%259F%258C%259E-Atlas-Design-Library%3Fnode-id%3D1488%253A35182
+classType: Component
+classPrefixes:
+  - accordion
+---
+
+# Accordion
+
+The Accordion component contains a disclosure widget, where information is only visible when the element is toggled to an "open" state. A Summary or label must be provided using the `<summary>` element.
+
+## Usage
+
+Here is an example of a standard `.accordion` usage. By default `<details>` elements are closed on page load and toggle open or closed when the user clicks the `<summary>`.
+
+```html
+<details class="accordion">
+	<summary>Accordion header 1</summary>
+	<div class="accordion-content">
+		<p>Accordion panel 1</p>
+	</div>
+</details>
+<details class="accordion">
+	<summary>Accordion header 2</summary>
+	<div class="accordion-content">
+		<p>Accordion panel 2</p>
+	</div>
+</details>
+<details class="accordion">
+	<summary>Accordion header 3</summary>
+	<div class="accordion-content">
+		<p>Accordion panel 3</p>
+	</div>
+</details>
+```
+
+## Attributes
+
+### Open
+
+There is an option to have the accordion component load in the expanded state by adding the `open` attribute.
+
+```html
+<details class="accordion" open>
+	<summary>Accordion header 1</summary>
+	<div class="accordion-content">
+		<p>Accordion panel 1</p>
+	</div>
+</details>
+<details class="accordion">
+	<summary>Accordion header 2</summary>
+	<div class="accordion-content">
+		<p>Accordion panel 2</p>
+	</div>
+</details>
+<details class="accordion">
+	<summary>Accordion header 3</summary>
+	<div class="accordion-content">
+		<p>Accordion panel 3</p>
+	</div>
+</details>
+```
+
+### Name
```

Comment:
In Firefox `name` doesn't work =\ need to figure out a work around this

Reply:
Looks like they released a fix for this, it isn't in the latest 129.0.2 but will be included in v130. Not sure when that is scheduled to be released.
https://bugzilla.mozilla.org/show_bug.cgi?id=1856460
https://github.com/mdn/content/pull/35447

Reply:
Found it, being release Sept 3 🎉
https://developer.mozilla.org/en-US/docs/Mozilla/Firefox/Releases/130

Reply:
Haha lucky you! Still need to keep an eye on it and verify it works soon. Otherwise, we have to figure out another way to make it work consistently throughout all the browsers

Reply:
@jwmunn @ollips It's also worth keeping in mind that it'll take some time for enough Firefox users to update to support `name` - see ["Evergreen" Does Not Mean Immediately Available](https://css-tricks.com/evergreen-does-not-mean-immediately-available/) - and so even when Firefox releases, we shouldn't necessarily *rely* on `name` to guarantee exclusivity.

That said, I think it's worth calling out that we can use `name` as a progressive enhancement -- if a browser version supports `name`, great, but if it doesn't, we wouldn't consider that "broken."

Which brings me to another point, which is should we put in guidance about when exclusivity is appropriate? I think Atlas should *discourage* exclusivity unless there's a really compelling reason why specifically only one `<details>` can be open at a time (where tidiness/compactness would not be a satisfactory reason).

If we had such guidance, it would also be appropriate to add a note about progressive enhancement in there.

Reply:
Both of the existing implementations currently in docs-ui allow multiple panels to be open at once, I agree that "name" can be a progressive enhancement.

Reply:
Confirmed it now works in the production latest firefox release 🎉